src/GovUk.Education.ExploreEducationStatistics.Admin/Security/AuthorizationHandlers/ViewSpecificReleaseAuthorizationHandlers.cs

```diff
@@ -1,123 +1,32 @@
 
-namespace GovUk.Education.ExploreEducationStatistics.Admin.Security.AuthorizationHandlers
-{
-    public class ViewSpecificReleaseAuthorizationHandler : CompoundAuthorizationHandler<ViewReleaseRequirement, Release>
-    {
-        public ViewSpecificReleaseAuthorizationHandler(
-            IPreReleaseService preReleaseService,
-            AuthorizationHandlerResourceRoleService authorizationHandlerResourceRoleService) 
-            : base(
-                new CanSeeAllReleasesAuthorizationHandler(),
-                new HasOwnerOrApproverRoleOnParentPublicationAuthorizationHandler(authorizationHandlerResourceRoleService),
-                new HasUnrestrictedViewerRoleOnReleaseAuthorizationHandler(authorizationHandlerResourceRoleService),
-                new HasPreReleaseRoleWithinAccessWindowAuthorizationHandler(preReleaseService, authorizationHandlerResourceRoleService))
-        {
-        }
-
-        public class CanSeeAllReleasesAuthorizationHandler : HasClaimAuthorizationHandler<
-            ViewReleaseRequirement>
-        {
-            public CanSeeAllReleasesAuthorizationHandler()
-                : base(SecurityClaimTypes.AccessAllReleases)
-            {
-            }
-        }
-        
-        public class HasUnrestrictedViewerRoleOnReleaseAuthorizationHandler
-            : AuthorizationHandler<ViewReleaseRequirement, Release>
-        {
-            private readonly AuthorizationHandlerResourceRoleService _authorizationHandlerResourceRoleService;
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Security.AuthorizationHandlers;
 
-            public HasUnrestrictedViewerRoleOnReleaseAuthorizationHandler(
-                AuthorizationHandlerResourceRoleService authorizationHandlerResourceRoleService)
-            {
-                _authorizationHandlerResourceRoleService = authorizationHandlerResourceRoleService;
-            }
-
-            protected override async Task HandleRequirementAsync(
-                AuthorizationHandlerContext context,
-                ViewReleaseRequirement requirement,
-                Release release)
-            {
-                if (await _authorizationHandlerResourceRoleService
-                        .HasRolesOnRelease(
-                            context.User.GetUserId(),
-                            release.Id,
-                            UnrestrictedReleaseViewerRoles))
-                {
-                    context.Succeed(requirement);
-                }
-            }
-        }
-
-        public class HasOwnerOrApproverRoleOnParentPublicationAuthorizationHandler
-            : AuthorizationHandler<ViewReleaseRequirement, Release>
-        {
-            private readonly AuthorizationHandlerResourceRoleService _authorizationHandlerResourceRoleService;
+public class ViewSpecificReleaseAuthorizationHandler : AuthorizationHandler<ViewReleaseRequirement, Release>
+{
+    private readonly IPreReleaseService _preReleaseService;
```

Comment:
I don't think an `IPreReleaseService` is required here as the only check being made is `_authorizationHandlerService.IsReleaseViewableByUser`

Reply:
Good spot - removed, thanks